csrc/cutlass_extensions/vllm_numeric_conversion.cuh

```diff
@@ -0,0 +1,787 @@
+// Based off of:
+//   https://github.com/NVIDIA/cutlass/blob/cutlass-3.5.0/include/cutlass/numeric_conversion.h
```

Comment:
How close to the upstream `numeric_conversion.h` is this file? I grepped around with varying success with the names in this file so it was a little hard to know. 

If this is very close to `cutlass/numeric_conversion.h`, then when we should turn off formatting and try to keep it as close to the upstream as possible, so it's easy to keep in sync with upstream changes. 

In any case, could you add a block comment describing what this file does, how it differs from the upstream, and why it's needed?

Reply:
so this was bad copy and pasting, this file is purely an extension of `numeric_conversion.h`, added a block comment